recipes/chado-tools/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "chado-tools" %}
+{% set version = "0.0.3" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "e8b7407013eee815288589f07314655d1e6ffc7c688f7daf58080fc4369a3668" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sanger-pathogens/{{ name }}/archive/v{{ version }}.{{ file_ext }}
+  {{ hash_type }}: {{ hash_value }}
+
+build:
+  number: 0
+  noarch: generic
```

Comment:
I guess this can be `noarch: python`?

Reply:
I'm not sure. It's a command line tool, and according to [the documentation](https://bioconda.github.io/linting.html#should-be-noarch), it should then be "generic" - if I understood correctly.

Reply:
Tests on my machine pass flawlessly with either setting. Let me know what you think is better.

Reply:
python is preferred I guess.

Reply:
Ok, thanks. Changed it.